content/livesync/Setup

```diff
@@ -0,0 +1,190 @@
+---
+title: SDK setup
+meta_description: "Install, authenticate and instantiate the Ably Realtime Data Models SDK."
+product: LiveSync
+---
+
+Use these instructions to install, authenticate and instantiate the Ably Realtime Data Models SDK.
+
+h2(#authenticate). Authenticate
+
+An "API key":/auth#api-keys is required to authenticate with Ably. API keys are used either to authenticate directly with Ably using "basic authentication":/auth/basic, or to generate tokens for untrusted clients using "token authentication":/auth/token.
+
+<aside data-type='important'>
+<p>The examples use "basic authentication":/auth/basic to demonstrate features for convenience. In your own applications, basic authentication should never be used on the client-side as it exposes your Ably API key. Instead use "token authentication":/auth/token. </p>
+</aside>
+
+"Sign up":https://ably.com/sign-up to Ably to create an API key in the "dashboard":https://ably.com/dashboard or use the "Control API":/account/control-api to create an API programmatically.
+
+API keys and tokens have a set of capabilities assigned to them that specify which operations, such as @subscribe@ or @publish@ can be performed on which resources. To use the Ably Realtime Data Models SDK, the API key requires the following "capabilities":/auth/capabilities#58: 
+
+* Publish
+* Subscribe
+* Presence
+* History
```

Comment:
The repo only seems to state subscribe is required.

Reply:
Subscribe capability is the only hard requirement

The Models SDK can optionally be used with history, see https://github.com/ably-labs/models/blob/main/docs/replay.md#history